{{cookiecutter.repository_name}}/setup.py

```diff
@@ -5,10 +5,10 @@
 
 from setuptools import setup, find_packages
 
-with open('README.md') as readme_file:
+with open('README.md', encoding="utf-8") as readme_file:
```

Comment:
nice

Reply:
can you use single quotes for consistency within this file?

Reply:
yep